cypress/utils/utils.ts

```diff
@@ -1627,13 +1627,13 @@ export function disableSwitch(selector: string): void {
 export function validateTooShortInput(selector, anotherSelector?: string): void {
     inputText(selector, "ab");
     if (anotherSelector) click(anotherSelector);
-    doesExistText("This field must contain at least 3 characters.", true);
+    doesExistText("Name is invalid. The name must be between 3 and 120 characters", true);
```

Comment:
This will fail if the message hasn't change in all places.

You can add a parameter to override the default message.